### PR TITLE
Add Ansible integration tests

### PR DESCRIPTION
**Pull request by lukash**
With a separate Dockerfile for now.

It's in a WIP status, but I'm planning to merge this, integrate it into the CI and go from there...

### CODE REVIEW

File: Dockerfile.ansible [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Build
+# -----
+# $ podman build -f Dockerfile.ansible -t ansible
+#
+# Run
+# ---
+# $ podman run -it --rm ansible-base ./ansible
```
[the remark this answers is not in the document]
Comment (lukash) [reply Dockerfile.ansible:7]: Right, thanks.

**Comment by lukash** [comment]
> Given that I'm leaving for PTO tomorrow and the tests are working, I would merge this. May be we should somehow better organize Dockerfiles. There is starting to be a lot of them right in the root directory.

I'm planning to attempt that, once I got the CI part down and know what's feasible on that side...

In general AFAIU it's a problem to share code between Dockerfiles, you can do a `FROM`, but that's from an image, so you need to somehow make that consistent...

I have an idea of doing something like:
```
ARG BASE=fedora:31
FROM $BASE
```
Then we could parametrize the base image, for running on multiple OSes (as opposed to copy-pasting the whole file) as well as potentially chain Dockerfiles. But at this stage I'm not sure it will work...

**Comment by lukash** [comment]
@m-blaha I've fixed the inconsistency and I've also added a commit changing the maintainer in Jenkins - I've already tested updating the jobs on Jenkins on that.

**Comment by m-blaha** [comment]
> Then we could parametrize the base image, for running on multiple OSes (as opposed to copy-pasting the whole file) as well as potentially chain Dockerfiles. But at this stage I'm not sure it will work...

This could work at least for Fedoras. Maybe having some kind of Dockerfile generator might also help. Instead of `podman build -f Dockerfile` use something like `generate_docker_file | podman build -` (somehow incorporated to container-test script)

**Comment by lukash** [comment]
> ... Maybe having some kind of Dockerfile generator might also help. Instead of `podman build -f Dockerfile` use something like `generate_docker_file | podman build -` (somehow incorporated to container-test script)

I very much hope we won't have to resort to that :slightly_smiling_face:

**Pull request merged**